model/_global/delete_pit/structures.smithy

```diff
@@ -21,6 +21,7 @@ list PitIds{
 }
 
 
+@documentation("a point-in-time id to delete")
```

Comment:
Lets keep the documentation consistent. One example: https://github.com/opensearch-project/opensearch-api-specification/blob/037843f96c36349b943514ffb59198d9c0855296/model/indices/put_alias/structures.smithy#L23C1-L23C1. May be we can write "The point-in-time ids to be deleted".